test/test_judiVector.jl

```diff
@@ -20,6 +20,11 @@ ftol = 1e-6
     info = example_info(nsrc=nsrc)
     dsize = (nsrc*nrec*ns, 1)
     rec_geometry = example_rec_geometry(nsrc=nsrc, nrec=nrec)
+    for i = 1:nsrc
```

Comment:
I think the problem here is that there is a rounding in the locations due to working w/ SegyIO. If we just do a `==` test on `judiVector` then it will return false because of incompatible geometry